draft-ietf-quic-transport.md

```diff
@@ -2682,6 +2681,10 @@ ack-eliciting packet if no other ack-eliciting packets have been sent since last
 receiving and processing a packet. Restarting this timer when sending a packet
 ensures that connections are not closed after new activity is initiated.
 
+There is no minimum value for max_idle_timeout.  To avoid excessively small idle
```

Comment:
This is like a major nitpick, but isn't the minimum value actually 1ms, because 0 means no timeout?